Sources/Sentry/SentryScreenshot.m

```diff
@@ -47,7 +47,12 @@ - (void)saveScreenShots:(NSString *)path
 
         if ([window drawViewHierarchyInRect:window.bounds afterScreenUpdates:false]) {
             UIImage *img = UIGraphicsGetImageFromCurrentImageContext();
-            [result addObject:UIImagePNGRepresentation(img)];
+            if (img.size.width > 0 || img.size.height > 0) {
+                NSData *bytes = UIImagePNGRepresentation(img);
+                if (bytes && bytes.length > 0) {
```

Comment:
`l`: If the image has 0 height or width, it's already empty. I think we don't need to check for `bytes && bytes.length > 0` here.

Reply:
Its not the size of the view that makes the image 0 length, its not being able to render the image.